init.rb

```diff
@@ -50,7 +50,7 @@
 
 #This check is needed because the overrides file will override the FLAVOR selection.
 #Furthermore a selection != stable can cause a inconsistent layout (cause by in_flavor system in the package_sets)
-if File.exists?(File.join(Autoproj.root_dir, "autoproj", "overrides.d", "25-release.yml")) && current_flavor.branch != "stable" 
+if ENV['ROCK_RC'] != '1' && File.exists?(File.join(Autoproj.root_dir, "autoproj", "overrides.d", "25-release.yml")) && current_flavor.branch != "stable" 
```

Comment:
:+1:. I've done something that looks like that. Could you check whether the message makes sense to you ?